app/javascript/dashboard/assets/scss/_variables.scss

```diff
@@ -67,6 +67,14 @@ $masked-bg: rgba(0, 0, 0, .4);
 $color-primary-light: #c7e3ff;
 $color-primary-dark: darken($color-woot, 20%);
 
+// Social and inboxes brand colors
+$facebook-brand-color: #3b5998;
+$twitter-brand-color: #1da1f2;
+$whatsapp-brand-color: #25d366;
+$sms-twilio-color: #f42f46;
+$chat-company-brand-color: $primary-color;
+$cloud-generic-color: #18b7b0;
```

Comment:
If possible can you use a color in `app/javascript/shared/assets/stylesheets/colors.scss` ?

Reply:
As I said earlier in my previous comment I purposefully did not add it in shared because all the colors used here are only relevant in use for the dashboard. None of the colors I have declared here have used outside of the dashboard. For example, the Facebook blue declared as $facebook-brand-color has no use or usage in any of the other projects under chatwoot like widgets etc